torchdynamo/config.py

```diff
@@ -86,7 +86,11 @@
 
 # If a PyTorch module is in this allowlist, torchdynamo will be allowed
 # to inline objects from it or its children.
-skipfiles_inline_module_allowlist = {torch.nn, torch.distributions}
+skipfiles_inline_module_allowlist = {
+    torch.nn,
+    torch.distributions,
+    torch.testing,
```

Comment:
we can also patch this in during tests, if we don't want it here, but I do not think it matters much.